www/js/diary/list.js

```diff
@@ -763,23 +783,11 @@ angular.module('emission.main.diary.list',['ui-leaflet',
           $rootScope.barDetail = false;
           }
         if($rootScope.displayingIncident == true) {
-          if (angular.isDefined(Timeline.data.currDay)) {
+          const tripProp = $rootScope.notificationData;
```

Comment:
Here we load the timeline based on the `start_ts` of notification data to get the diary page display the correct day.